inference-engine/src/mkldnn_plugin/emitters/emitter.cpp

```diff
@@ -2,10 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "emitter.h"
-
+#include "emitter.hpp"
```

Comment:
As we discussed offline: please rename `emit()`->`emit_code()` and `emit_table()`->`emit_data()`